blocks/feed-newsroom/feed-newsroom.css

```diff
@@ -0,0 +1,89 @@
+/* eslint no-empty: "info" */
+
+main .section.feed-newsroom-container > .section-container {
+    max-width: 1200px;
+}
+
+main .section.feed-newsroom-container.narrow {
+    max-width: 90rem;
+}
+
+main .block.feed-newsroom {
+    gap: 10vw;
+}
+
+main .block.feed-newsroom > .others:first-child {
+    flex-grow: 3;
+}
+
+main .feed-newsroom-wrapper > div:first-child {
```

Comment:
Styles applied on `wrapper` would also influence other blocks within this wrapper, do we want that?

Reply:
Removed that. Fixed